corehq/apps/case_importer/templates/case_importer/partials/ko_import_status.html

```diff
@@ -0,0 +1,64 @@
+{% load i18n %}
+<!--ko if: state === $root.states.SUCCESS-->
+    <div class="text-success">
+        <!--ko if: result.match_count-->
+            <p>
+                {% blocktrans with match_count='result.match_count' %}
+                    <strong data-bind="text: {{ match_count }}"></strong> rows were matched and
+                    updated.
+                {% endblocktrans %}
+            </p>
+        <!--/ko-->
+
+        <!--ko if: result.created_count > 0-->
+            <p>
+                {% blocktrans with created_count='result.created_count' %}
```

Comment:
If these are available in the template (`result.created_count`) why use knockout to determine whether or not to show them? Especially since they don't seem to get updated by any JS if the value changes.